src/adapters/supabase/helpers/comment.ts

```diff
@@ -28,11 +28,10 @@ export class Comment extends SuperSupabase {
     //First Check if the comment already exists
     const { data, error } = await this.supabase.from("issue_comments").select("*").eq("id", commentNodeId);
     if (error) {
-      this.context.logger.error("Error creating comment", error);
-      return;
+      throw new Error("Error creating comment");
     }
     if (data && data.length > 0) {
-      this.context.logger.info("Comment already exists");
```

Comment:
You should be able to just throw this, then it logs and halts function execution.

Reply:
@gentlementlegen rfc

Reply:
@gentlementlegen rfc

Reply:
You can just `throw logger.fatal` so you get the log displayed and the error caught on a higher level of execution.

Reply:
Probably should be error